provider-announcements/src/java/org/sakaiproject/archiver/provider/AnnouncementsArchiver.java

```diff
@@ -192,6 +165,17 @@ private ArchiveItem createArchiveItem(final AnnouncementMessage announcement) {
 		@Setter
 		private String createdOn;
 
+		@Getter
+		@Setter
+		private String attachments = "";
+
+		public SimpleAnnouncement(final AnnouncementMessage announcement) {
+			setTitle(announcement.getAnnouncementHeader().getSubject());
```

Comment:
generally it would be `this.title = announcement.getAnnouncementHeader().getSubject()` unless the setter had some special stuff in it.